foreman.te

```diff
@@ -385,23 +381,16 @@ read_files_pattern(websockify_t, puppet_var_lib_t, puppet_var_lib_t)
 # Container / Docker
 #
 
-optional_policy(`
-    tunable_policy(`passenger_can_connect_docker_tcp',`
-        ifdef(`has_container', `
-            container_stream_connect(passenger_t)
-        ')
-        ifdef(`has_docker', `
-            docker_stream_connect(passenger_t)
-        ')
-    ')
-')
+allow passenger_t foreman_container_port_t:tcp_socket name_connect;
 
 optional_policy(`
-    tunable_policy(`passenger_can_connect_docker_unix',`
+    tunable_policy(`passenger_can_connect_container_unix',`
         ifdef(`has_container', `
+            container_stream_connect(passenger_t)
             container_spc_stream_connect(passenger_t)
         ')
         ifdef(`has_docker', `
+            docker_stream_connect(passenger_t)
             #docker_spc_stream_connect(passenger_t)
             gen_require(`
                 type spc_t, spc_var_run_t;
```

Comment:
Renamed the boolean and added missing `*_stream_connect` - this was actually a bug, it was in the `tcp` block but in does belong here.